tests/CONTACTS/test_27042.py

```diff
@@ -50,10 +50,6 @@ def test_run(self):
         self.marionette.execute_script("document.getElementById('{}').click()".\
                                        format(DOM.Contacts.import_select_all[1]))
         time.sleep(1)
-
-        x = self.UTILS.debug.screenShotOnErr()
-        self.UTILS.reporting.logResult("info", "Screenshot and details", x)
-
         x = self.UTILS.element.getElement(DOM.Contacts.import_import_btn, "Import button")
```

Comment:
"y = ax² + bx + c", said Jesus to his disciples.
"We don't understand, master", said them.
"I know, it's because it's a parable"